data-source/jsonnet/england-wales/ccs/blocks/who-lives-here/relationships.jsonnet

```diff
@@ -1,6 +1,7 @@
 {
   type: 'Question',
   id: 'relationships',
+  show_on_section_summary: false,
   question: {
     id: 'relationships-question',
     title: 'Are any of these people related to each other? Remember to include partners and step-children as related.',
```

Comment:
This can now include the list_summary as the feature has been merged to v3.

Reply:
We've been asked to include this as part of CCS v3 and not this PR, so will go into the next PR.